spyder/plugins/application/container.py

```diff
@@ -284,73 +279,99 @@ def _check_updates_ready(self):
-        box.set_checked(check_updates)
+        box.set_checked(self.get_conf(option))
+
+        header = _(f"<b>Spyder {latest_release} is available!</b> "
+                   f"<i>(you have {__version__})</i><br><br>")
 
         if error_msg is not None:
-            msg = error_msg
-            box.setText(msg)
+            box.setText(error_msg)
             box.set_check_visible(False)
             box.exec_()
-            check_updates = box.is_checked()
-        else:
-            if update_available:
-                if self.application_update_status:
-                    self.application_update_status.set_status_pending(
-                        latest_release=latest_release)
 
-                header = _("<b>Spyder {} is available!</b> "
-                           "<i>(you have {})</i><br><br>").format(
-                    latest_release, __version__)
-                content = ""
-                footer = _(
-                    "For more information, visit our "
-                    "<a href=\"{}\">installation guide</a>."
-                ).format(url_i)
-                if is_anaconda():
-                    content = _(
-                        "<b>Important note:</b> Since you installed "
-                        "Spyder with Anaconda, please <b>don't</b> use "
-                        "<code>pip</code> to update it as that will break "
-                        "your installation.<br><br>"
-                        "Instead, run the following commands in a "
-                        "terminal:<br>"
-                        "<code>conda update anaconda</code><br>"
-                        "<code>conda install spyder={}</code><br><br>"
-                    ).format(latest_release)
-                elif is_pynsist() or running_in_mac_app():
-                    box.setStandardButtons(QMessageBox.Yes |
-                                           QMessageBox.No)
-                    content = _(
-                        "Would you like to automatically download and "
-                        "install it?<br><br>"
-                    )
-
-                msg = header + content + footer
+        elif update_available:
+            # Update using our installers
+            if (
+                not sys.platform.startswith("linux")
+                or parse(latest_release) >= parse("6.0.0")
+            ):
+                box.setStandardButtons(QMessageBox.Yes | QMessageBox.No)
+                box.setDefaultButton(QMessageBox.Yes)
+
+                msg = (
+                    header +
+                    _("Would you like to automatically download "
+                      "and install it? <br><br>")
+                )
+
+                if not is_pynsist() and not running_in_mac_app():
+                    msg += _("Clicking Yes will download and install the "
+                             "latest version using Spyder's installer, leaving"
+                             " your existing installation untouched.<br><br>")
```

Comment:
```suggestion
                    msg += _("Clicking <b>Yes</b> will download and install the "
                             "latest version using Spyder's installer, leaving "
                             "your existing installation untouched.<br><br>")
```

* Make the `Yes` answer different from the rest of the text (we could use italics if you prefer, but I think we should do this in any case).
* Add space at the end of strings because it's easier to spot it (we try to do that in all our multi-line strings).